reference-implementation/lib/transform-stream.js

```diff
@@ -88,7 +92,7 @@ function TransformStreamError(stream, e) {
 }
 
 function TransformStreamSetBackpressure(stream, backpressure) {
-  // console.log(`TransformStreamSetBackpressure(${backpressure})`);
+  verbose(`TransformStreamSetBackpressure(${backpressure})`);
```

Comment:
Maybe `(stream, ${backpressure})` or `() [backpressure = ${backpressure}]` or similar? Right now the mismatch seems a bit confusing.

Reply:
I went with the latter as it seems odd to have this method log argument names when the others don't.